### PR TITLE
Zhilan Fang HW01

### PR DESCRIPTION
**Pull request by zlfang97**
I am Zhilan Fang, a MAPSS Sociology student. For this homework, I wrote a short introduction of myself using headers, lists, emphasis, links and an image. I have never used Github or R markdown before, but I find the homework workflow video very helpful in getting me started. I also find the guidance on markdown cheatsheet very easy to follow. I met a problem when I tried to insert a picture in README. I uploaded my picture to my GitHub repository and pasted that address link to my markdown. However, when I tried to push the change, I got rejected several times. I later googled the problem and realized that I had to pull from my repository first, then I succeeded. I also tried to add a caption for my picture because I thought this was quite common. I googled this and also searched on Stack Overflow, but I don't think I've found a right way. I eventually failed and deleted the caption. Maybe Professor Soltoff and TAs have a solution to it. Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
 # hw01
+
+## **About me**
+Hi! My name is **Zhilan Fang**. I'm a MAPSS student with a concentration in Sociology. I'm from China and will stay in [UChicago Beijing Center](https://www.uchicago.cn/) during the fall quarter. I'm broadly interested in topics about *inequality, social media, and opinion formation*. I'm looking forward to learning more computational skills in this course and applying them to my studies.
+
+## **Other things you might want to know**
+* Hold a bachelor's degree in **Journalism** from [Tsinghua University](https://www.tsinghua.edu.cn/en/index.htm)
```
Comment (whiskey0504) [inline README.md:7]: my roommate is from Tsinghua too haha

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+* Love watching films and documentaries, and also had experience shooting a short documentary about **Tibetan folk songs**
```
Comment (whiskey0504) [inline README.md:8]: this sounds so cool.......

**Review by whiskey0504 — COMMENTED** [review]
impeccable. Used headers, emphasis (both bold and italic), lists, image, and links to websites, perfectly fulfilling the requirement of the assignment. If I were a cat person who graduated from Peking University, I might wanna nitpick something out of this--but I'm not.

File: README.md [hunk 3]
```diff
@@ -1 +1,15 @@
...
+* Definitely a dog person
+* Also love cooking but is still trying to improve my cooking skills
+
+Here is a photo of me:
+
+![profile](https://github.com/zlfang97/hw01/blob/master/profile.jpg?raw=true)
```
Comment (whiskey0504) [inline README.md:14]: profile
*ok I tried to google it as well and I found that probably the easiest way to do it is just to insert an italic line like this down below*

Comment (deblnia) [inline README.md:14]: Because this picture is included in your repo, you can use a relative link like so: 
```
![profile](profile.jpg)
```

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |           |      X        |            |
| **Presentation**            |   X        |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Reproducibility**         |   X        |              |            |

Remarks:

* Overall, great work! You've used a ton of markdown syntax, everything seems to be rendered correctly, and you seem comfortable with the git submission workflow, so that's great! 
* I wouldn't worry about captions now -- they're a bit difficult to do in plain markdown, but we'll look at how to caption something in a few weeks when we learn about R Markdown. 
* I'd also try and use relative links -- the absolute link (`https...`) is a bit platform dependent, and it's generally good strategy for a repo to be self contained. Happy to talk about this more in office hours if anything's unclear!

File: README.md [hunk 4]
```diff
@@ -1 +1,15 @@
 # hw01
+
+## **About me**
+Hi! My name is **Zhilan Fang**. I'm a MAPSS student with a concentration in Sociology. I'm from China and will stay in [UChicago Beijing Center](https://www.uchicago.cn/) during the fall quarter. I'm broadly interested in topics about *inequality, social media, and opinion formation*. I'm looking forward to learning more computational skills in this course and applying them to my studies.
```
Comment (jnseddon) [inline README.md:4]: Nice, my friend is also doing is MA in Sociology here right now!

File: README.md (hunk 3 above)
Comment (jnseddon) [inline README.md:14]: I'm repeating what @deblnia said, but yep, relative links are your friend! I struggled with that in this assignment too.

File: README.md [hunk 5]
```diff
@@ -1 +1,15 @@
...
+
+## **Other things you might want to know**
```
Comment (jnseddon) [inline README.md:6]: I like the structure of this list!

**Review by jnseddon — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       +    |              |            |
| **Coding strategy**         |      +     |              |            |
| **Presentation**            |       +    |              |            |
| **Achievement, creativity** |   +        |              |            |
| **Reproducibility**         |      +     |              |            |

Remarks:

I like your introduction a lot! I like how you presented yourself in the list. There's not anything I can say for criticism, but getting relative links down might be a helpful skill to pick up. As I mentioned, I struggled with that too.

**Pull request closed without merging**